2014-10-23 17:16:02,592 [main ] INFO  net.waqassiddiqi.app.crew.ui.control.RibbonbarTabControl - Initializing RibbonbarTabControl
2014-10-23 17:16:02,694 [main ] DEBUG net.waqassiddiqi.app.crew.db.ConnectionManager - executing: SELECT * FROM app_settings
2014-10-23 17:16:03,497 [main ] DEBUG net.waqassiddiqi.app.crew.db.ConnectionManager - executing: SELECT * FROM app_settings
2014-10-23 17:16:05,513 [main ] ERROR net.waqassiddiqi.app.crew.db.DatabaseClient - Server: jdbc:h2:tcp://192.168.1.3:9090/./resource/restdb;IFEXISTS=TRUE is not available
org.h2.jdbc.JdbcSQLException: Connection is broken: "java.net.SocketTimeoutException: connect timed out: 192.168.1.3:9090" [90067-181]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:450)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:327)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at net.waqassiddiqi.app.crew.db.DatabaseClient.isServerAvailable(DatabaseClient.java:18)
	at net.waqassiddiqi.app.crew.ui.MainFrame.<init>(MainFrame.java:163)
	at net.waqassiddiqi.app.crew.ui.MainFrame.getInstance(MainFrame.java:85)
	at net.waqassiddiqi.app.crew.ui.MainFrame.runApplication(MainFrame.java:459)
	at net.waqassiddiqi.app.crew.ui.MainFrame.main(MainFrame.java:508)
Caused by: java.net.SocketTimeoutException: connect timed out
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:122)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:102)
	at org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:112)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:446)
	... 11 more
2014-10-23 17:16:13,069 [SwingWorker-pool-1-thread-1 ] ERROR net.waqassiddiqi.app.crew.db.DatabaseClient - Server: jdbc:h2:tcp://192.168.1.3:9090/./resource/restdb;IFEXISTS=TRUE is not available
org.h2.jdbc.JdbcSQLException: Connection is broken: "java.net.SocketTimeoutException: connect timed out: 192.168.1.3:9090" [90067-181]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:450)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:327)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at net.waqassiddiqi.app.crew.db.DatabaseClient.isServerAvailable(DatabaseClient.java:18)
	at net.waqassiddiqi.app.crew.ui.MainFrame$5$1.doInBackground(MainFrame.java:268)
	at net.waqassiddiqi.app.crew.ui.MainFrame$5$1.doInBackground(MainFrame.java:1)
	at javax.swing.SwingWorker$1.call(SwingWorker.java:296)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at javax.swing.SwingWorker.run(SwingWorker.java:335)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: java.net.SocketTimeoutException: connect timed out
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:122)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:102)
	at org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:112)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:446)
	... 15 more
2014-10-23 18:06:51,899 [main ] INFO  net.waqassiddiqi.app.crew.ui.control.RibbonbarTabControl - Initializing RibbonbarTabControl
2014-10-23 18:06:52,007 [main ] DEBUG net.waqassiddiqi.app.crew.db.ConnectionManager - executing: SELECT * FROM app_settings
2014-10-23 18:06:55,878 [main ] ERROR net.waqassiddiqi.app.crew.db.DatabaseClient - Server: jdbc:h2:tcp://127.0.0.1:9090/./resource/restdb;IFEXISTS=TRUE is not available
org.h2.jdbc.JdbcSQLException: Connection is broken: "java.net.ConnectException: Connection refused: connect: 127.0.0.1:9090" [90067-181]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:450)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:327)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at net.waqassiddiqi.app.crew.db.DatabaseClient.isServerAvailable(DatabaseClient.java:18)
	at net.waqassiddiqi.app.crew.ui.MainFrame.<init>(MainFrame.java:167)
	at net.waqassiddiqi.app.crew.ui.MainFrame.getInstance(MainFrame.java:86)
	at net.waqassiddiqi.app.crew.ui.MainFrame.runApplication(MainFrame.java:467)
	at net.waqassiddiqi.app.crew.ui.MainFrame.main(MainFrame.java:516)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:122)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:102)
	at org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:112)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:446)
	... 11 more
2014-10-23 18:07:30,061 [main ] INFO  net.waqassiddiqi.app.crew.ui.control.RibbonbarTabControl - Initializing RibbonbarTabControl
2014-10-23 18:07:49,571 [main ] INFO  net.waqassiddiqi.app.crew.ui.control.RibbonbarTabControl - Initializing RibbonbarTabControl
2014-10-23 18:07:53,315 [main ] ERROR net.waqassiddiqi.app.crew.db.DatabaseClient - Server: jdbc:h2:tcp://127.0.0.1:9090/./resource/restdb;IFEXISTS=TRUE is not available
org.h2.jdbc.JdbcSQLException: Connection is broken: "java.net.ConnectException: Connection refused: connect: 127.0.0.1:9090" [90067-181]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:450)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:327)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at net.waqassiddiqi.app.crew.db.DatabaseClient.isServerAvailable(DatabaseClient.java:18)
	at net.waqassiddiqi.app.crew.ui.MainFrame.<init>(MainFrame.java:167)
	at net.waqassiddiqi.app.crew.ui.MainFrame.getInstance(MainFrame.java:86)
	at net.waqassiddiqi.app.crew.ui.MainFrame.runApplication(MainFrame.java:467)
	at net.waqassiddiqi.app.crew.ui.MainFrame.main(MainFrame.java:516)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:122)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:102)
	at org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:112)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:446)
	... 11 more
2014-10-23 18:08:38,507 [main ] INFO  net.waqassiddiqi.app.crew.ui.control.RibbonbarTabControl - Initializing RibbonbarTabControl
2014-10-23 18:08:42,212 [main ] ERROR net.waqassiddiqi.app.crew.db.DatabaseClient - Server: jdbc:h2:tcp://127.0.0.1:9090/./resource/restdb;IFEXISTS=TRUE is not available
org.h2.jdbc.JdbcSQLException: Connection is broken: "java.net.ConnectException: Connection refused: connect: 127.0.0.1:9090" [90067-181]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:450)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:327)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at net.waqassiddiqi.app.crew.db.DatabaseClient.isServerAvailable(DatabaseClient.java:18)
	at net.waqassiddiqi.app.crew.ui.MainFrame.<init>(MainFrame.java:167)
	at net.waqassiddiqi.app.crew.ui.MainFrame.getInstance(MainFrame.java:86)
	at net.waqassiddiqi.app.crew.ui.MainFrame.runApplication(MainFrame.java:467)
	at net.waqassiddiqi.app.crew.ui.MainFrame.main(MainFrame.java:516)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:122)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:102)
	at org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:112)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:446)
	... 11 more
2014-10-23 18:08:55,964 [SwingWorker-pool-1-thread-1 ] ERROR net.waqassiddiqi.app.crew.db.DatabaseClient - Server: jdbc:h2:tcp://192.168.1.3:9090/./resource/restdb;IFEXISTS=TRUE is not available
org.h2.jdbc.JdbcSQLException: Connection is broken: "java.net.SocketTimeoutException: connect timed out: 192.168.1.3:9090" [90067-181]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:450)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:327)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at net.waqassiddiqi.app.crew.db.DatabaseClient.isServerAvailable(DatabaseClient.java:18)
	at net.waqassiddiqi.app.crew.ui.MainFrame$5$1.doInBackground(MainFrame.java:273)
	at net.waqassiddiqi.app.crew.ui.MainFrame$5$1.doInBackground(MainFrame.java:1)
	at javax.swing.SwingWorker$1.call(SwingWorker.java:296)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at javax.swing.SwingWorker.run(SwingWorker.java:335)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: java.net.SocketTimeoutException: connect timed out
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:122)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:102)
	at org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:112)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:446)
	... 15 more
2014-10-23 18:11:02,524 [main ] INFO  net.waqassiddiqi.app.crew.ui.control.RibbonbarTabControl - Initializing RibbonbarTabControl
2014-10-23 18:11:04,710 [main ] ERROR net.waqassiddiqi.app.crew.db.DatabaseClient - Server: jdbc:h2:tcp://127.0.0.1:9090/./resource/restdb;IFEXISTS=TRUE is not available
org.h2.jdbc.JdbcSQLException: Connection is broken: "java.net.ConnectException: Connection refused: connect: 127.0.0.1:9090" [90067-181]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:450)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:327)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at net.waqassiddiqi.app.crew.db.DatabaseClient.isServerAvailable(DatabaseClient.java:18)
	at net.waqassiddiqi.app.crew.ui.MainFrame.<init>(MainFrame.java:167)
	at net.waqassiddiqi.app.crew.ui.MainFrame.getInstance(MainFrame.java:86)
	at net.waqassiddiqi.app.crew.ui.MainFrame.runApplication(MainFrame.java:467)
	at net.waqassiddiqi.app.crew.ui.MainFrame.main(MainFrame.java:516)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:122)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:102)
	at org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:112)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:446)
	... 11 more
2014-10-23 18:11:23,882 [main ] INFO  net.waqassiddiqi.app.crew.ui.control.RibbonbarTabControl - Initializing RibbonbarTabControl
2014-10-23 18:11:25,986 [main ] ERROR net.waqassiddiqi.app.crew.db.DatabaseClient - Server: jdbc:h2:tcp://127.0.0.1:9090/./resource/restdb;IFEXISTS=TRUE is not available
org.h2.jdbc.JdbcSQLException: Connection is broken: "java.net.ConnectException: Connection refused: connect: 127.0.0.1:9090" [90067-181]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:450)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:327)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at net.waqassiddiqi.app.crew.db.DatabaseClient.isServerAvailable(DatabaseClient.java:18)
	at net.waqassiddiqi.app.crew.ui.MainFrame.<init>(MainFrame.java:167)
	at net.waqassiddiqi.app.crew.ui.MainFrame.getInstance(MainFrame.java:86)
	at net.waqassiddiqi.app.crew.ui.MainFrame.runApplication(MainFrame.java:467)
	at net.waqassiddiqi.app.crew.ui.MainFrame.main(MainFrame.java:516)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:122)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:102)
	at org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:112)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:446)
	... 11 more
2014-10-23 18:12:36,528 [main ] INFO  net.waqassiddiqi.app.crew.ui.control.RibbonbarTabControl - Initializing RibbonbarTabControl
2014-10-23 18:12:38,654 [main ] ERROR net.waqassiddiqi.app.crew.db.DatabaseClient - Server: jdbc:h2:tcp://10.0.100.83:9090/./resource/restdb;IFEXISTS=TRUE is not available
org.h2.jdbc.JdbcSQLException: Connection is broken: "java.net.ConnectException: Connection refused: connect: 10.0.100.83:9090" [90067-181]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:450)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:327)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at net.waqassiddiqi.app.crew.db.DatabaseClient.isServerAvailable(DatabaseClient.java:18)
	at net.waqassiddiqi.app.crew.ui.MainFrame.<init>(MainFrame.java:167)
	at net.waqassiddiqi.app.crew.ui.MainFrame.getInstance(MainFrame.java:86)
	at net.waqassiddiqi.app.crew.ui.MainFrame.runApplication(MainFrame.java:467)
	at net.waqassiddiqi.app.crew.ui.MainFrame.main(MainFrame.java:516)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:122)
	at org.h2.util.NetUtils.createSocket(NetUtils.java:102)
	at org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:112)
	at org.h2.engine.SessionRemote.connectServer(SessionRemote.java:446)
	... 11 more
